packages/server/integrations/TaskIntegrationManagerFactory.ts

```diff
@@ -40,11 +41,13 @@ export interface TaskIntegrationManager {
     teamDashboardUrl: string,
     integrationHash: string
   )
+
+  getIssue?(taskId: string)
 }
 
 export default class TaskIntegrationManagerFactory {
   public static async initManager(
-    dataLoader: DataLoaderWorker,
+    dataLoader: DataLoaderWorker | RootDataLoader,
```

Comment:
-2 this looks wrong, but gets resolved if you don't call this factory from the dataloader anymore